tokio/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.2.17 (April 9, 2020)
+
+### Fixes
+- rt: bug in work-stealing queue (#2387) 
```

Comment:
Other commits to master since 0.2.16 were docs and test fixes, so I didn't add them to the changelog.